[2023-12-20T11:49:18.457+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_id scheduled__2022-01-30T00:00:00+00:00 [queued]>
[2023-12-20T11:49:18.476+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_id scheduled__2022-01-30T00:00:00+00:00 [queued]>
[2023-12-20T11:49:18.477+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2023-12-20T11:49:18.602+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): second_id> on 2022-01-30 00:00:00+00:00
[2023-12-20T11:49:18.613+0000] {standard_task_runner.py:60} INFO - Started process 5731 to run task
[2023-12-20T11:49:18.626+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_id', 'scheduled__2022-01-30T00:00:00+00:00', '--job-id', '1350', '--raw', '--subdir', 'DAGS_FOLDER/first_dat.py', '--cfg-path', '/tmp/tmppx02su2k']
[2023-12-20T11:49:18.631+0000] {standard_task_runner.py:88} INFO - Job 1350: Subtask second_id
[2023-12-20T11:49:18.792+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.second_id scheduled__2022-01-30T00:00:00+00:00 [running]> on host 29b3723064ac
[2023-12-20T11:49:19.173+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abrham' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_id' AIRFLOW_CTX_EXECUTION_DATE='2022-01-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-01-30T00:00:00+00:00'
[2023-12-20T11:49:19.175+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-20T11:49:19.176+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be runing after task 3']
[2023-12-20T11:49:19.239+0000] {subprocess.py:86} INFO - Output:
[2023-12-20T11:49:19.268+0000] {subprocess.py:93} INFO - hey, I am task2 and will be runing after task 3
[2023-12-20T11:49:19.269+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-20T11:49:19.378+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_id, execution_date=20220130T000000, start_date=20231220T114918, end_date=20231220T114919
[2023-12-20T11:49:19.577+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2023-12-20T11:49:19.633+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
